Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    gris (0.3.8)
+    gris (0.3.9)
```

Comment:
Looks like in your merge from artsy/master the version dropped back to 0.3.8 :/ https://github.com/bhoggard/gris/commit/adb4e91b1b2cb6e49885eef6bd583dbe9bd978d9

I'll merge and then update the version and cut a release.